### PR TITLE
LPS-145967 Move previewURL calculation to D&M info API

### PR DESCRIPTION
**Pull request by cgoncas**
Motivation
=======
We are using themeDisplay and LiferayPortletURL in Content Dashboard API (these are not correct parameters for an API)
[LPS-145967](https://issues.liferay.com/browse/LPS-145967)

Proposed Solution
========
Move previewURL code to the Document&Media Info API and remove LiferayPortletRequest and ThemeDisplay as parameters

Steps to Verify:
----------------
1.This is not a change with funtional impact so all Content Dashboard Info Panel tests should work as in the previous versions.

### CODE REVIEW

**Comment by cgoncas** [comment]
Hey @liferay-lima !!!

I would love if you can check how we have included previewURL in the DL info API (https://github.com/liferay-lima/liferay-portal/pull/2678/commits/5c51da2400f4f6d746978f47ce42b6d7280d67f9)

Thanks!!

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/info/item/provider/FileEntryInfoItemFieldValuesProvider.java [hunk 1]
```diff
@@ -276,6 +276,16 @@ private String _getDisplayPageURL(
 						FileEntryInfoItemFields.downloadURL, downloadURL));
 			}
 
+			String previewURL = _dlURLHelper.getPreviewURL(
+				fileEntry, fileEntry.getFileVersion(), themeDisplay,
+				StringPool.BLANK, false, true);
+
+			if (Validator.isNotNull(downloadURL)) {
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/info/item/provider/FileEntryInfoItemFieldValuesProvider.java:283]: Shouldn't this be `Validator.isNotNull(previewURL)`?

**Review by adolfopa — CHANGES_REQUESTED** [review]
Other than the typo it looks good. Can you push force the fix?

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/info/item/provider/FileEntryInfoItemFieldValuesProvider.java (hunk 1 above)
Comment (cgoncas) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/info/item/provider/FileEntryInfoItemFieldValuesProvider.java:283]: That is totally right :)

Comment (cgoncas) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/info/item/provider/FileEntryInfoItemFieldValuesProvider.java]: Fixed and pushed.

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**